ccw.core/src/java/ccw/core/StaticStrings.java

```diff
@@ -0,0 +1,9 @@
+package ccw.core;
+
+public class StaticStrings {
```

Comment:
Would you mind renaming StaticStrings class to something carrying more semantics, for instance SystemPropertiesConstants ?